src/index.html

```diff
@@ -1,17 +1,159 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>The form</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      method="POST"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="fs1">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          minlength="3"
+          maxlength="20"
+          autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+          type="text"
+          name="surname"
+          minlength="3"
+          maxlength="20"
+          autocomplete="off"
+          >
+        </label>
+        <label class="label">
```

Comment:
add an empty line between _multiline_ siblings
```suggestion
        </label>
        
        <label class="label">
```